trace/trace.go

```diff
@@ -38,12 +38,22 @@ func StartSpan(c context.Context, operation string, spanOpts ...sentry.SpanOptio
 		return c, func() {}
 	}
 
-	functionName := "unknown function"
-	if pc, _, _, ok := runtime.Caller(1); ok {
-		functionName = runtime.FuncForPC(pc).Name()
+	if len(spanOpts) == 0 {
+		// Add defalut options if none provided.
+		functionName := "unknown function"
+		if pc, _, _, ok := runtime.Caller(1); ok {
+			functionName = runtime.FuncForPC(pc).Name()
+		}
+		spanOpts = append(spanOpts, sentry.WithDescription(functionName))
+	}
+
+	parent := sentry.SpanFromContext(c)
```

Comment:
Is there any logic to determine whether the parent span exists? I remember that `StartSpan` will determine it internally.

Reply:
Yeah, I also read the method and there is no change in functionality here, but I think this would be a bit easier to understand.